src/ui/mathliveview.js

```diff
@@ -0,0 +1,35 @@
+import InputView from '@ckeditor/ckeditor5-ui/src/view';
+import { Template } from '@ckeditor/ckeditor5-ui';
+
+export default class MathLiveView extends InputView {
+	constructor( locale, { options, attributes, style } ) {
+		super( locale );
+		this._options = options;
+		const bind = Template.bind( this, this );
+		this.set( 'value', '' );
+		this.setTemplate( {
+			tag: 'math-field',
+			attributes: {
+				...attributes,
+				style
+			},
+			on: {
+				input: bind.to( event => {
+					this.value = this.element.value;
+					this.fire( 'input', event );
+				} )
```

Comment:
Based on https://ckeditor.com/docs/ckeditor5/latest/api/module_ui_template-TemplateListenerSchema.html, I think it could be also:
```suggestion
				input: [
					bind.to( event => this.value = this.element.value ),
					bind.to( 'input' )
				]
```

Probably doesn't make a practical difference, but looks nicer IMO 🙂